src/lib/GOBLBuilder.svelte

```diff
@@ -27,12 +36,24 @@
   // generated and used for signing GOBL documents.
   export let signEnabled = true;
 
+  // When enabled, actions buttons such as Build, Sign and Validate are hidden. The underlying actions
+  // that they fire can be still executed from outside.
+  export let hideActionButtons = false;
+
+  // Menu instance to be able to call functions from outside
+  let menu: MenuBar;
```

Comment:
It does feel strange having a menu bar with hidden actions... would it make sense to have the actions defined somewhere explicitly without them being UI components?